src/js/components/VoterGuide/OrganizationCard.jsx

```diff
@@ -137,16 +137,45 @@ export default class OrganizationCard extends Component {
       if (this.state.organizationPosition.vote_smart_rating) {
```

Comment:
I would recommend pulling `organizationPosition` out of the state variable above this if/else block like this first:
```const { organizationPosition } = this.state;```
Then below, you don't need to write out `this.state.organizationPosition` multiple times.